src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const population = [...document.body.querySelectorAll('.population')]
+  .map(item => +item.textContent.replace(/,/g, ''));
+const totalPopulation = population.reduce((sum, current) => sum + current, 0);
+const averagePopulation = totalPopulation / population.length;
+
+document.body.querySelector('.total-population').textContent
+  = getThousandsSeparator(totalPopulation, ',');
+
+document.body.querySelector('.average-population').textContent
+  = getThousandsSeparator(averagePopulation, ',');
```

Comment:
try this 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString